### PR TITLE
DOC-7519 -- Pending Doc Ids not documented

### PR DESCRIPTION
**Pull request by ibsoln**
DOC-7519 -- Pending Doc Ids not documented

https://issues.couchbase.com/browse/DOC-7519

### CODE REVIEW

**Comment by ibsoln** [comment]
Can you review the code in the below commits please? I added a snippet to show PendindDocIDs and IsDocumentPending in action.

- @bmeike  -- JAVA Snippets … https://github.com/couchbase/docs-couchbase-lite/pull/408/commits/3a57b76686a6bf62cc0cf9222e2a449b35e7f73d
- @jayahariv   -- iOS Snippets  https://github.com/couchbase/docs-couchbase-lite/pull/408/commits/425fabb79f2463cdeb32e0e583d667a3c6472c9a 
- @Sandychuang8  -- C# Snippets … https://github.com/couchbase/docs-couchbase-lite/pull/408/commits/400daa55755d8600151c16c5495944fd7f4380ac

File: modules/objc/examples/code_snippets/SampleCodeTest.m [hunk 1]
```diff
@@ -836,6 +836,66 @@ - (void) dontTestReplicatorStatus {
...
+        [replicator addChangeListener:^(CBLReplicatorChange *change) {
+
+          NSLog(@"Replicator activity level is %u", change.status.activity);
+          // iterate and report-on the pending doc IDs  in 'mydocids'
+
+          NSEnumerator *enumerator =
+            [mydocids objectEnumerator];
+          id thisid;
+
+          while (thisid =[ enumerator nextObject]) {
```
Comment (jayahariv) [inline modules/objc/examples/code_snippets/SampleCodeTest.m]: probably more easier?
```
for (thisid in mydocids) {
  NSError* err = nil;
  if (![replicator isDocumentPending: thisid error: &err]) { // <.>
    NSLog(@"Doc ID %@ now pushed", thisid);
  }
}

**Review by jayahariv — APPROVED** [review]
looks ok

File: modules/objc/examples/code_snippets/SampleCodeTest.m (hunk 1 above)
Comment (ibsoln) [reply modules/objc/examples/code_snippets/SampleCodeTest.m]: Thanks @jayahariv

**Pull request merged**